src/xmoto/SysMessage.cpp

```diff
@@ -230,17 +230,18 @@ void SysMessage::render_console() {
   if ((m_consoleLastShowTime + SYSMSG_CONSOLEDISPLAY_TIME +
            SYSMSG_CONSOLEDISPLAY_ANIMATIONTIME >
          v_time ||
-       XMSession::instance()->permanentConsole()) &&
+       XMSession::instance()->permanentConsole() ||
+       StateManager::instance()->isThereASuchState("StateMessageBox")) &&
       NetClient::instance()->isConnected()) {
     if ((m_consoleLastShowTime + SYSMSG_CONSOLEDISPLAY_TIME > v_time ||
-         XMSession::instance()->permanentConsole()) &&
+         XMSession::instance()->permanentConsole() ||
+       StateManager::instance()->isThereASuchState("StateMessageBox")) &&
```

Comment:
Really not a fan of this nested if statement, but it seems to play a relevant role.
Sigh.